package.json

```
@@ -27,18 +27,14 @@
     "start": "react-app-rewired start",
     "build": "react-app-rewired build",
     "test": "react-app-rewired test",
-    "lint": "npm run lint:editorconfig && npm run lint:eslint",
-    "lint:editorconfig": "editorconfig-checker",
-    "lint:eslint": "eslint --ext .js,.jsx,.ts,.tsx src",
-    "lint:eslint:fix": "npm run lint:eslint -- --fix",
+    "lint:eslint": "eslint src --ext .ts,.tsx",
+    "lint:prettier": "prettier --check \"src/**/*.{js,jsx,ts,tsx,json}\"",
+    "lint": "npm-run-all lint:*",
```

```suggestion
    "lint": "run-p lint:*",
```
Using `run-p` here makes it clearer they're running in parallel (npm-run-all's default behaviour)